discovery_windows.go

```diff
@@ -28,6 +28,7 @@ import (
 // see https://github.com/composer/windows-setup/blob/master/src/composer.iss
 func (s *PHPStore) doDiscover() {
 	systemDir := systemDir()
+	userProfileDir := userProfileDir()
```

Comment:
what about `os.UserHomeDir` instead ?

Reply:
I was aware of it, but assumed in the context of Windows it was essentially just a proxy for getting the environment variable. Though I am confused as to why MAMP is handled in this method. Happy for it to change to UserHomeDir if there's more going on here or if you feel it's more appropriate.

Reply:
Oh there's a MAMP for Windows? I had no idea.

Reply:
On balance it is probably more correct to use UserHomeDir. If a future version of Windows changes something we can rely on it being correct in Go upstream. Want me to do another PR @tucksaun?

Reply:
I've added a new PR to change this

Reply:
> If a future version of Windows changes something we can rely on it being correct in Go upstream.

yes this was my thinking